mintpy/save_geodmod.py

```diff
@@ -0,0 +1,166 @@
+  save_geodmod.py $SCRATCHDIR/DarbandikhanSenAT72/PYSAR/INPUTS/ifgramStack.h5 20180603_20180615 -b 34.2 35.2 45.0 46.3 -y 0.001 -x 0.001 -t cor -o AT20180603_20180615.cor
+  save_geodmod.py $SCRATCHDIR/DarbandikhanSenAT72/PYSAR/temporalCoherence.h5 20171117_20180603 -b 34.2 35.2 45.0 46.3 -y 0.001 -x 0.001 -t cor -o AT20171117_20180603.cor
+  save_geodmod.py $SCRATCHDIR/DarbandikhanSenAT72/PYSAR/INPUTS/geometryRadar.h5 20171117_20180603 -b 34.2 35.2 45.0 46.3 -y 0.001 -x 0.001 -t dem -o AT.dem
+
+"""
+
+def create_parser():
+    parser = argparse.ArgumentParser(description='Get horizontal and vertical displacements based on ascending and descending data',
+                                     formatter_class=argparse.RawTextHelpFormatter,
+                                     epilog=EXAMPLE)
+
+    parser.add_argument('file', nargs=1, help='ascending and descending timeseries files\n')
+    parser.add_argument('dset', nargs=1, help='date12 of timeseries to be converted')
+    parser.add_argument('-b', '--bbox', dest='SNWE', type=float, nargs=4, metavar=('S', 'N', 'W', 'E'),
+                        help='Bounding box of area to be geocoded.\n' +
+                        'Include the uppler left corner of the first pixel' +
+                        '    and the lower right corner of the last pixel')
+    parser.add_argument('-y', '--latstep', dest='latStep', type=float,
+                        help='output pixel size in degree in latitude.')
+    parser.add_argument('-x', '--lonstep', dest='lonStep', type=float,
+                        help='output pixel size in degree in longitude.')
+    parser.add_argument('-t','--type',dest='Type',type=str,
+                        help='The type of data with choice [unw, cor, dem]')
+    parser.add_argument('-o', '--output', dest='outfile', nargs=1,
+                        help="output file name.")
+    return parser
+
+
+def cmd_line_parse(iargs=None):
+    parser = create_parser()
+    inps = parser.parse_args(args=iargs)
+    return inps
+    
+def format_args(arr, dst=""):
+    """ parse list array(list item or values) to string """
+    for k in arr:
+        if type(k) == list:
+            dst = dst + " " + format_args(k)
+        else:
+            dst = dst + " " + str(k)
+    return dst.strip()
+   
+def get_path_com(path):
+    """ return directory(filepath), filename(shotname), extension """
+    (filepath, tempfilename) = os.path.split(os.path.abspath(path))
+    (filename, extension) = os.path.splitext(tempfilename)
+    return filepath, filename, extension
+
+def write_rsc_file(in_file,out_file):
+    """ write rsc file for Geodmod just estract several properities from rsc file"""
+    # read file
+    meta = readfile.read_roipac_rsc(in_file)
+    # initiate dict
+    rsc = dict()
+    rsc['FILE_DIR'] = 'lxr/Iraq/afterslip_example'
```

Comment:
I recommend you to initiate this value based on the input file, hardwired value is not recommended in general. I am not sure if this metadata is important or not, @falkamelung knows it better.

Reply:
Thank you for your suggestions and I modified this part.